journal/202105.md

```diff
@@ -0,0 +1,475 @@
+
+- updated [About Politeia](https://github.com/decred/politeiagui/pull/2378) copy
+- put proposal token into email notification [subject](https://github.com/decred/politeia/pull/1417)
+- consistently handle [short](https://github.com/decred/politeiagui/pull/2365) proposal tokens across the UI codebase
+- fixed multiple [comment](https://github.com/decred/politeiagui/pull/2384)-related UI bugs
+- fixed [flickering](https://github.com/decred/politeiagui/pull/2395) during initial UI loading
+- ~2 other backend and ~14 UI bug fixes
+- fixed [unit](https://github.com/decred/politeiagui/pull/2364) and [end-to-end](https://github.com/decred/politeiagui/pull/2383) tests to work with the new backend API
+- dependency upgrades, code cleanup, API enhancement
+
+Contractor Management System (CMS):
+
+- changed invoice peer review [visibility](https://github.com/decred/politeia/pull/1351) from based on author's domain to based on invoice's line item domain. This will allow contractors to review all billed work in their domain even if it is coming from contractors in other domains. The current domains are: Development, Research, Design and Marketing.
+- added more email [notifications](https://github.com/decred/politeia/pull/1353): "Timely submission of invoices will be much more important once the payout from the treasury is fully automated, so further nagging is due."
+- docs update and code cleanup
+- ~4 backend and ~1 UI bug fixes
+
+Politeia v1.0.1 has been [released](https://github.com/decred/politeia/releases/tag/v1.0.1) including all the above backend fixes and improvements.
+
+Progress towards v1.1.0 can now be tracked on the [Milestones](https://github.com/decred/politeia/milestones) page.
+
+<a id="vspd" />
+
+**[vspd](https://github.com/decred/vspd)**
+
+- added block explorer [links](https://github.com/decred/vspd/pull/247) for ticket and fee transactions to the UI
+- added best block [height](https://github.com/decred/vspd/pull/254) to status response (useful for detecting stalled VSPs)
+- populate ticket [purchase height](https://github.com/decred/vspd/pull/250) for all confirmed tickets and show it on the admin page
+- improved admin page [aesthetics](https://github.com/decred/vspd/pull/263)
+- added a framework for database [upgrades](https://github.com/decred/vspd/pull/242)
+- improved logging when [recovering](https://github.com/decred/vspd/pull/255) from request handling failures
+- added [config](https://github.com/decred/vspd/pull/249) params for max log size and max log files to keep
+- [remove](https://github.com/decred/vspd/pull/260) confirmed fee transactions from the database, cutting disk usage [by ~2x](https://github.com/decred/vspd/pull/243#issuecomment-844945549)
+- store each ticket in its own [DB bucket](https://github.com/decred/vspd/pull/243). Combined with the previous optimization, it allows to trade ~40% higer disk usage for ~50% faster insertion and ~85% faster counting of tickets.
```

Comment:
Not just counting tickets, but any operation which requires iterating over all tickets - eg. searching/filtering. Maybe go with "~85% faster iterating over tickets."?

Reply:
That will work, changed. Thanks for the suggestion.